lib/util/mongo.js

```diff
@@ -177,10 +177,20 @@ function hasUserSpaceDb() {
   return false;
 }
 
+function buildConnectionString(dbConf, mongoHost, replicaSet) {
+  var connString = "mongodb://" + dbConf.user + ":" + dbConf.pass + "@" + mongoHost + ":" + dbConf.port + "/" + dbConf.name;
+  if (replicaSet) {
+    connString +=  "?replicaSet=" + replicaSet;
+  }
```

Comment:
Since it's here already i guess it's fine, but we don't intend to add this replicaSet option on core component mongo connection strings.

Reply:
It handles both scenarios (replica set or not) so I decided to leave it as it is.